src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AssessmentReferralHistoryNoteTransformer.kt

```diff
@@ -40,6 +44,16 @@ class AssessmentReferralHistoryNoteTransformer {
     else -> throw RuntimeException("Unsupported ReferralHistoryNote type: ${jpa::class.qualifiedName}")
   }
 
+  fun transformToReferralHistoryDomainEventNote(updatedEvent: CAS3AssessmentUpdatedEvent, user: UserEntity) =
+    ReferralHistoryDomainEventNote(
+      id = updatedEvent.id,
+      createdAt = updatedEvent.timestamp,
+      message = "",
```

Comment:
is it possible to make this type nullable in the schema to better reflect this may happen? I guess in this case the UI will use the message details to build a message?